Main.as

```diff
@@ -39,15 +46,17 @@ void RenderEarly()
 
 		mat4 projection = mat4::Perspective(camFov, camAspect, camNearZ, camFarZ);
 		mat4 translation = mat4::Translate(vec3(camLoc.tx, camLoc.ty, camLoc.tz));
-		mat4 rotation = mat4::Inverse(mat4::Inverse(translation) * mat4(camLoc));
+		mat4 inverseTranslation = mat4::Inverse(translation);
+		g_rotation = mat4::Inverse(inverseTranslation * camLocMat);
```

Comment:
Probably doesn't need to be 2 lines, and could be grouped together with the `g_projection` and `g_position` assignments.

Reply:
done